app/react/components/LoggedInNav.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+export default (props) => {
+  return (
+    <nav className="mdl-navigation">
+      {props.auth.get('role') === 'admin' ? <Link to="admin-dashboard" className="mdl-navigation__link">All Cases</Link> : ''}
+      {props.auth.get('role') === 'admin' ? <Link to="advocates" className="mdl-navigation__link">Advocates</Link> : ''}
+      {props.auth.get('role') === 'admin' ? <Link to="volunteers" className="mdl-navigation__link">Volunteers</Link> : ''}
```

Comment:
Yeah, I was thinking that as well. Probably the easiest way of going about this would be to do a giant `props.auth.get('role')` and then rendering all of it in a div instead of doing it individually.